apis/r/R/SOMADataFrame.R

```diff
@@ -42,6 +43,16 @@ SOMADataFrame <- R6::R6Class(
       # Parse the tiledb/create/ subkeys of the platform_config into a handy,
       # typed, queryable data structure.
       tiledb_create_options <- TileDBCreateOptions$new(platform_config)
+      #print(str(tiledb_create_options$to_list()))
+
+      ## TODO: the private$.soma_context object contains an xptr to a struct containtaining
```

Comment:
please file a GitHub issue to track, or Shortcut story, and include its link here

Reply:
Actually I will look into removing it from `create()` and nix the comment with it.   This was from a first pass, it is now clearer that no `create()` method should be needed.  

Done in 091e1094 

Nice catch!